Gemfile

```diff
@@ -2,7 +2,7 @@ source "http://rubygems.org"
 
 gemspec
 
-gem 'geometry-in-ruby', github: 'aurorasolar/geometry', tag: 'v0.0.6'
+gem 'geometry-in-ruby', github: 'aurorasolar/geometry', tag: 'v1.0.0'
```

Comment:
nice! I guess it wouldn't be beneficial to start publishing those as full private gems right now?